app/views/junks/show.html.erb

```diff
@@ -1,99 +1,91 @@
-            <div>
-              <div class="label-name"><p>City</p></div>
-              <div class="boxed">
-                <p><%= @junk.address.split(", ")[1].split[1] %></p>
+          <div class="junk-show-form-info">
+            <div class="price-favourites">
+              <div>
+                <p><%= @junk.price %>€</p>
               </div>
-            </div>
 
-            <div>
-              <div class="label-name"><p>Postal code</p></div>
-              <div class="boxed">
-                <p><%= @junk.address.split(", ")[1].split[0] %></p>
+              <div id="favourites">
+              <%= image_tag "heart.svg" %>
+              <p>Add to Favourites</p>
               </div>
             </div>
 
-            <div class="delivery">
-            <%= image_tag "delivery.svg" %>
-            <p>Delivery possible</p>
+            <div class="address">
+              <div>
+                <div class="label-name"><p>City</p></div>
+                <div class="boxed">
+                  <p><%= @junk.address.split(", ")[1].split[1] %></p>
+                </div>
+              </div>
+
+              <div>
+                <div class="label-name"><p>Postal code</p></div>
+                <div class="boxed">
+                  <p><%= @junk.address.split(", ")[1].split[0] %></p>
+                </div>
+              </div>
+
+              <div class="delivery">
+              <%= image_tag "delivery.svg" %>
+              <p>Delivery possible</p>
+              </div>
             </div>
-          </div>
 
-          <div class="description">
-            <div class="label-name"><p>Description</p></div>
-            <div class="junk-description-scrollable">
-              <p><%= @junk.description %></p>
+            <div class="description">
+              <div class="label-name"><p>Description</p></div>
+              <div class="junk-description-scrollable">
+                <p><%= @junk.description %></p>
+              </div>
             </div>
           </div>
+
+
         </div>
 
 
+        <div class="junk-show-images-button">
+          <div class="small-image-container">
+            <% @junk.photos.first(6).each do |photo| %>
+            <%= cl_image_tag(photo.key, class: 'small-image', alt: 'image junk') %>
+            <% end %>
+          </div>
+          <%= link_to "Contact seller", junks_path, class: "primary-button-1" %>
       </div>
+          <div class="map">
+            <%= render "shared/map"%>
+          </div>
 
-
-      <div class="junk-show-images-button">
-        <div class="small-image-container">
-          <% @junk.photos.first(6).each do |photo| %>
-          <%= cl_image_tag(photo.key, class: 'small-image', alt: 'image junk') %>
-          <% end %>
-        </div>
-        <%= link_to "Contact seller", junks_path, class: "primary-button-1" %>
+      </div>
     </div>
-        <div class="map">
-          <%= render "shared/map"%>
-        </div>
 
-    </div>
   </div>
-  </section>
```

Comment:
hmmm. not sure about this as in my version there is a <section> and </section> wrap around the map. It seems the <section> is not there but everything removed in final version. I think this is ok.

Reply:
idk what the purpose of this <section  is but it had the class main-container and it was f up with the overall layout of the page. 

If section is important then you can add it again proving it doesnt change anytbing again, and we have one main-container css in the application.scss file and we must not use this class elsewhere